### ISSUE TITLE
Sass compiler continuously restarting

### ISSUE THREAD

**Issue by DatReki** [koenvzeijl/AspNetCore.SassCompiler#23]
When I installed this package it continuously kepts restarting.
This is what it produces in my console (just keeps outputting the same thing)
```
info: AspNetCore.SassCompiler.SassCompilerHostedService[0]
      Started Sass watch
info: AspNetCore.SassCompiler.SassCompilerHostedService[0]
      Positional and ":" arguments may not both be used.
info: AspNetCore.SassCompiler.SassCompilerHostedService[0]
      Usage: sass <input.scss> [output.css]
info: AspNetCore.SassCompiler.SassCompilerHostedService[0]
             sass <input.scss>:<output.css> <input/>:<output/> <dir/>
info: AspNetCore.SassCompiler.SassCompilerHostedService[0]
      === Input and Output ===================
info: AspNetCore.SassCompiler.SassCompilerHostedService[0]
          --[no-]stdin               Read the stylesheet from stdin.
info: AspNetCore.SassCompiler.SassCompilerHostedService[0]
          --[no-]indented            Use the indented syntax for input from stdin.
warn: AspNetCore.SassCompiler.SassCompilerHostedService[0]
      Sass compiler exited, restarting in 1 second.
```
![Image of the issue](https://cdn.discordapp.com/attachments/406006274661154818/892846412776235041/unknown.png)

I copied my project and removed everything till I was down to a basic ASP MVC template. Issue was still there. [I put it on my secondary GitHub account](https://github.com/DatRekiBurner/AspNetCore.SassCompiler-Restarting). 

However when I downloaded that repository to check if the issue still persisted it was suddenly gone. 
On the project that I shaved down to a basic MVC template (the code between that and the linked repo is the same) the issue still exists so I sadly cannot tell you what steps to take to reproduce it (obviously already cleared bin/obj).

All three projects use the latest version of ASP.Net Core (MVC) with .Net5

**Comment by sleeuwen** [comment]
Thanks for reporting the issue and creating the repo, is it possible you're first project has a space somewhere in the directory hierarchy but the test project does not?

We have had an issue before with spaces and fixed it during dotnet publish, but I just tested it and get the same error as you have using the compiler service. I've created a fix for this and released it as 1.42.1.1, can you try that and see if it fixes your issue?

**Comment by DatReki** [comment]
That does seem to indeed have fixed the issue for me. On both projects the issue disappeared.
